src/Aspire.Hosting/Dcp/IDcpDependencyCheckService.cs

```diff
@@ -35,4 +35,11 @@ internal sealed class DcpContainersInfo
 
     [JsonPropertyName("error")]
     public string? Error { get; set; }
+
+    internal string ContainerHostName =>
+        Runtime switch
+        {
+            "podman" => "host.containers.internal",
```

Comment:
@danegsta what do you think about making container host part of the DCP container runtime info?

Reply:
We should be returning the identified runtime as part of `dcp info` already.

Reply:
I like that idea

Reply:
@danegsta that issue is assigned to you already 😄

Reply:
DCP change to expose the address should be in PR in a few minutes